notes/stories/television/tv-theamericanshortstory.st.txt

```diff
@@ -229,3 +229,47 @@ sports [A group of friends were shown enjoying a leisurely game of croquet.]
 the game of tennis [Otis showed up at the house holding an old-timey, wooden framed tennis racket and was annoyed that his friend Charlie had forgotten about the game they had scheduled to play.]
 
 
+theamericanshortstory06
+=======================
+
+:: Title
+I'm a Fool
+
+:: Description
+A young horse trainer learns the hard way that telling a whopper of a lie is
+probably not the best way to win over the girl you fancy. It is based on the
+1923 Sherwood Anderson short story of the same name. 
+
+:: Date
+1977-04-05
+
+:: References
+https://www.imdb.com/title/tt0074662/
+
+:: Ratings
+4 <brent>
+4 <paul>
+
+:: Major Themes
+growing as a person [The story culminates with Andy realizing he was a fool for having tried to woo his love interest, Lucy, with lies of importance and wealth.]
+romantic love [Andy was smitten by Lucy, and Lucy was head over heels for the man who Andy feigned to be. Andy waxed poetic about how he wanted his first time to be with a princess.]
+mentor and protégé [Burt taught Andy the ins and outs of training race horses.]
+rite of passage into manhood [Andy went from being a young, bashful, and sexually inexperienced horse trainer to being a cigar-smoking, whiskey-drinking son of a gun, a man of the world and about town if you will, who was not above wooing girls at the horse races with lies of importance and wealth.]
+
+:: Minor Themes
+racism [Andy's family was ashamed of him working with a black man.]
+gender issue [Andy explained that he wanted to be the strong one in a relationship and for his future wife to be timid. Andy first introduced himself to Lucy's brother rather than speak to her directly.]
+music [Burt was fond of singing while they traveled from town to town.]
+fiancée and fiancée [Henry and Eleanor were an agreeable couple who seemed pleased that Lucy had a suitor.]
+friendship [Andy encouraged a drunken Burt to pursue his dream of being a jockey after practice one day. Burt gave Andy a great deal of useful life advice in a sometimes gruff, sometimes jolly manner.]
+brother and sister [Lucy West accompanied Henry and Eleanor to the race track.]
+gambling [Andy and his new friends bet a lot of money on Andy's horse and won big.]
+lying [Andy told a whopper of a lie to woo his love interest, and ultimately came to regret it.]
```

Comment:
"growing as a person" comment makes me wonder if "lying" shouldn't be major..?

Reply:
Not an unreasonable suggestion as it was much featured in the last third or so of the story and was also relevant to to conclusion. I knocked it up.